handlers/main.yml

```diff
@@ -13,7 +13,19 @@
 
 - name: restart network
   nohup:
-    command: /etc/init.d/network restart
+    command: service network restart
```

Comment:
NACK. `service` is an OpenWrt `ash`ism from [service.sh](https://github.com/openwrt/openwrt/blob/master/package/base-files/files/lib/functions/service.sh).

Unlike with `systemd`, where functionality has been intentionally brought into `systemctl`, here the functionality all rests within `procd`. `service` realizes this by manually parsing arguments for the busybox `start-stop-daemon` applet, which is a less direct and is thus less expected to be stable.

Reply:
Oh, that was the maintainer (@gekmihesg), not me :^)

I like the pull request; I also think there should be more handlers available. Firewall is a great one to start with.

Reply:
What I meant there was using the `service` module instead of the `nohup` module. The existing handlers will interrupt the connection between the Ansible controller and the device, because the network (or wifi) gets shut down and brought up again. Ansible will fail because of that.
In the firewall case, this is unnecessary. You can just use the `service` module like this:
```yaml
- name: reload firewall
  service:
    name: firewall
    state: reloaded
  when: "'openwrt' in group_names"
```

This will give you much better feedback if something goes wrong. So my change request stays the same ;-)